open-policy-agent/README.md

```diff
@@ -83,3 +83,85 @@ For example, applying 'kubeadmin' temporary user removal validation policy
 $ oc apply -f https://raw.githubusercontent.com/openshift-4-compliance/openshift-4-compliance-automation/master/open-policy-agent/authentication-user-management/delete-kubeadmin/template.yaml
 $ oc apply -f https://raw.githubusercontent.com/openshift-4-compliance/openshift-4-compliance-automation/master/open-policy-agent/authentication-user-management/delete-kubeadmin/constraint.yaml
 ```
+
+## Applying policies using GitOps
+You can include in `dryrun` mode either all policies or as policy groups.
+You can change to `enforce` mode either all policies, as policy groups, or individually.
+
+
+You can't include specific policies in dry run, you can use the alternative workflow suggested at the end of the paragraph in [this link](https://kubectl.docs.kubernetes.io/faq/kustomize/eschewedfeatures/#removal-directives).
+
+
+
+### Import this kustomization
+#### By extending this project
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+# All policies
+- <PATH_TO_/open-policy-agent_FOLDER>
+# Specific groups
+- <PATH_TO_/open-policy-agent_FOLDER/policy-folder>
+...
+```
+
+#### By importing this kustomization to your repo
+```
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+# All policies
+- https://github.com/openshift-4-compliance/openshift-4-compliance-automation.git//open-policy-agent?ref=<TAG>
+# Specific groups
+- https://github.com/openshift-4-compliance/openshift-4-compliance-automation.git//open-policy-agent/policy-folder?ref=<TAG>
+...
+```
+
+### Modify this kustomization
+#### Enforce specific policies
+```
+...
+
+patchesJson6902:
+- target:
+    name: <POLICY_NAME>
+  patch: |-
+    - op: replace
+      path: "/spec/enforcementAction"
+      value: deny
+```
+
+#### Enforce a group of policies
+```
+...
+
+patchesJson6902:
+- target:
+    name: .*
+    group: constraints.gatekeeper.sh
+    version: v1beta1
+    labelSelector: policy-group=<GROUP_NAME>
+  patch: |-
+    - op: replace
+      path: "/spec/enforcementAction"
+      value: deny
+```
+
+#### Enforce all policies
+```
+...
+
+patchesJson6902:
+- target:
+    name: .*
+    group: constraints.gatekeeper.sh
+    version: v1beta1
+  patch: |-
+    - op: replace
+      path: "/spec/enforcementAction"
+      value: deny
+```
```

Comment:
@galshi I'd add an example of implementation using ArgoCD at the end - which configurations should be applied in the `Application` for both `template` and `constraint` resources to be created and not result in `constraint` creation failure due to kustomization issues.

I know it might be an overkill, but as for now, if a simple user tries to use the examples we provide, it'll fail. The issue should be covered, and a solution needs to be provided for a complete GitOps solution.